text/0000-argument-spread-splarguments.md

```diff
@@ -0,0 +1,81 @@
+- Start Date: 2019-06-05
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Forwarding Component Arguments with "Splarguments"
```

Comment:
I would replace "Splarguments" here with the actual syntax, `...arguments`, or maybe "Spreadable Arguments". It's better to avoid colloquial terms like this in the final RFC text (for instance, my original RFC for computed overriding referenced "clobbering" all over the place, but a lot of people were confused because they didn't know what that term meant).